### PR TITLE
Completed TC01

### PR DESCRIPTION
**Pull request by trangnhomanhcoi2601**

### CODE REVIEW

File: src/main/java/page_objects/BasePage.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        return DriverHelper.getDriver().findElement(tabLogin);
+    }
+
+    //Methods
+    public void goToLoginTab() {
+        getLoginTab().click();
+    }
+
+}
+
```
Comment (anhlvt) [inline src/main/java/page_objects/BasePage.java]: remove redundant line

File: src/main/java/page_objects/LoginPage.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        return getLblWelcomeUserMessage().getText();
+    }
+
+    public void login(String user, String password) {
+        getTxtUsername().sendKeys(user);
+        getTxtPassword().sendKeys(password);
+        getBtnLogin().click();
+    }
+}
+
```
Comment (anhlvt) [inline src/main/java/page_objects/LoginPage.java]: remove redundant line

File: src/main/java/page_objects/BasePage.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    //Elements
+    private WebElement getLoginTab() {
+        return DriverHelper.getDriver().findElement(tabLogin);
+    }
+
+    //Methods
+    public void goToLoginTab() {
+        getLoginTab().click();
+    }
+
```
Comment (tanmle) [inline src/main/java/page_objects/BasePage.java]: redudant

File: src/main/java/page_objects/LoginPage.java (hunk 2 above)
Comment (tanmle) [inline src/main/java/page_objects/LoginPage.java]: redudant

File: src/main/java/page_objects/BasePage.java (hunk 1 above)
Comment (trangnhomanhcoi2601) [reply src/main/java/page_objects/BasePage.java]: I already fixed it.

File: src/main/java/page_objects/BasePage.java (hunk 3 above)
Comment (trangnhomanhcoi2601) [reply src/main/java/page_objects/BasePage.java]: I already fixed it.

File: src/main/java/page_objects/LoginPage.java (hunk 2 above)
Comment (trangnhomanhcoi2601) [reply src/main/java/page_objects/LoginPage.java]: I already fixed it.

File: src/main/java/helpers/DriverHelper.java [hunk 4]
```diff
@@ -26,8 +26,6 @@ public static void quit() {
     public static WebDriver getDriver() {
         return driver;
     }
-
-
 }
```
Comment (MinhQuan119) [inline src/main/java/helpers/DriverHelper.java:30]: remove redundant lines 30 to 33

File: src/main/java/page_objects/LoginPage.java [hunk 5]
```diff
@@ -0,0 +1,42 @@
+package page_objects;
+
+import helpers.DriverHelper;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class LoginPage extends BasePage {
+
```
Comment (MinhQuan119) [inline src/main/java/page_objects/LoginPage.java:8]: BasePage doesn't have this line
Please remove it or add it on BasePage to make they the same format

File: src/main/java/page_objects/LoginPage.java [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    }
+
+    public void login(String user, String password) {
+        getTxtUsername().sendKeys(user);
+        getTxtPassword().sendKeys(password);
+        getBtnLogin().click();
+    }
+}
+
+
```
Comment (MinhQuan119) [inline src/main/java/page_objects/LoginPage.java]: I still see empty redundant line at row 41 42, please remove them

File: src/main/java/helpers/DriverHelper.java (hunk 4 above)
Comment (trangnhomanhcoi2601) [reply src/main/java/helpers/DriverHelper.java:30]: i removed it

File: src/main/java/page_objects/LoginPage.java (hunk 6 above)
Comment (trangnhomanhcoi2601) [reply src/main/java/page_objects/LoginPage.java]: i removed it.

File: src/main/java/page_objects/LoginPage.java (hunk 5 above)
Comment (trangnhomanhcoi2601) [reply src/main/java/page_objects/LoginPage.java:8]: i fixed it. Could you help me check it again, thanks so much

**Review by anhlvt — APPROVED** [review]

**Review by tanmle — APPROVED** [review]

**Pull request merged**